keps/0024-parameter-enhancement.md

```diff
@@ -1,13 +1,13 @@
 ---
 kep-number: 24
-title: Enabling and Disabling features in KUDO Operators
+title: Enhanced Operator Parameters
 authors:
   - "@nfnt"
 owners:
   - "@nfnt"
 creation-date: 2020-02-21
 last-updated: 2020-02-21
```

Comment:
we should capture the `last-updated` on this

Reply:
Done.